### DISCUSSION TITLE
Train Pose model error

### DISCUSSION THREAD

**Discussion by TSUKAMOTO-Kenji** [ultralytics/ultralytics#2583] [Q&A]
Hi, mow I train a pose model. but I cannot train beacause I got a KeyError. 

```
File "C:\Project\Anaconda3\envs\YOLO\lib\site-packages\ultralytics\yolo\v8\pose\train.py", line 32, in get_model
    model = PoseModel(cfg, ch=3, nc=self.data['nc'], data_kpt_shape=self.data['kpt_shape'], verbose=verbose)
KeyError: 'kpt_shape'
```

**Comment by glenn-jocher** [comment]
@TSUKAMOTO-Kenji hello! It looks like you're encountering a `KeyError` because the expected key `'kpt_shape'` is missing from your dataset configuration. This key is required for pose estimation models as it defines the shape of the keypoints.

Please ensure that your dataset configuration file includes the `'kpt_shape'` entry with the correct dimensions for your keypoints. This entry should match the structure and format expected by the YOLOv8 pose estimation model.

If you need further guidance on configuring your dataset for pose estimation, please refer to the Pose/Keypoint Estimation section in our documentation. 😊📚
